scripts/irods/test/test_iadmin.py

```diff
@@ -60,6 +60,34 @@ def tearDown(self):
     # iadmin
     ###################
 
+    def test_iadmin_rmresc_failures__5545(self):
+        test_file = "foo"
+        lib.make_file(test_file, 50)
+
+        ptresc = "pt"
+        newresc = "newResc"
+        h = lib.get_hostname()
+
+        try:
+            # rmresc standalone resource, with data
+            self.admin.assert_icommand(['iadmin', 'mkresc', newresc, 'unixfilesystem', '{}:/tmp/{}Vault'.format(h,newresc)], 'STDOUT_SINGLELINE', 'Creating')
+            self.admin.assert_icommand(['iput', '-R', newresc, test_file])
+            self.admin.assert_icommand(['iadmin', 'rmresc', newresc], 'STDERR_SINGLELINE', 'CAT_RESOURCE_NOT_EMPTY')
+
+            # rmresc child resource, without data
+            self.admin.assert_icommand(['iadmin', 'mkresc', ptresc, 'passthru'], 'STDOUT_SINGLELINE', 'Creating')
+            self.admin.assert_icommand(['iadmin', 'addchildtoresc', ptresc, newresc])
+            self.admin.assert_icommand(['irm', '-f', test_file])
+            self.admin.assert_icommand(['iadmin', 'rmresc', newresc], 'STDERR_SINGLELINE', 'CHILD_EXISTS')
```

Comment:
Can you add one more to rmresc the parent?

Reply:
done

Reply:
and passes